include/wx/localedefs.h

```diff
@@ -108,6 +108,22 @@ struct WXDLLIMPEXP_BASE wxLanguageInfo
     const char* TrySetLocale() const;
 };
 
+// ----------------------------------------------------------------------------
+// wxLocaleIdent: contains platform dependent language name
+// ----------------------------------------------------------------------------
+
+class wxLocaleIdent
+{
+public:
+    wxLocaleIdent() = default;            // Leave name empty
+    wxLocaleIdent(const wxString& name);  // Constract name from wxString
+    wxString GetLocaleName() const;
+    bool IsDefault() const;               // Empty name represents user's default language
+private:
+    wxString LocaleName;                  // Platform dependent name, e.g.
```

Comment:
Should use the standard naming convention, i.e. `m_` prefix. I also think "locale" is redundant here, so why not just
```suggestion
    wxString m_name;                  // Platform dependent name, e.g.
```

However, as per above, I think this should actually be `m_language` and we should have a separate `m_region` etc.